README.md

```diff
@@ -373,6 +373,34 @@ CidType : 6
 
 ![MacCommand](/Docs/Pictures/MacCommand.PNG)
 
+## Updating exising installations
+
+## Updating existing installations from 0.3.0-preview to 0.4.0-preview
+
+### Updating IoT Edge Runtime Containers to Version 1.0.6 ###
+
+We highly recommend running the latest version of the IoT Edge runtime containers on your gateway to Version 1.0.6. The way that you update the `IoT Edge agent` and `IoT Edge hub` containers depends on whether you use rolling tags (like 1.0) or specific tags (like 1.0.2) in your deployment. 
+
+The process is outlined in detail [here](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-update-iot-edge#update-the-runtime-containers).
+
+Furthermore, make sure, the following environment variables are set for your `Edge hub` container:
+
+```
+mqttSettings__enabled: false
+httpSettings__enabled: false
+TwinManagerVersion: v2
+```
+
+You do this by clicking "Set Modules" &rarr; "Configure advanced edge runtime settings" on your IoT Edge device in Azure IoT Hub.
+
+Make sure the **DevAddr** of your ABP LoRa devices starts with **"02"**: Due to addition of NetId support in this pre-relese, ABP devices created by the template  prior to 0.4.0-preview (and all devices with an incompatible NetId in general) will be incompatible with the 0.4.0-preview. If you don't set a specific NetId, the default is 1. In this case, make sure the DevAddr of your ABP LoRa devices starts with "02".
```

Comment:
NetId is not user defined so it is currently fix to 1

Reply:
Changed.

Reply:
I would remove sentence "If you don't set a specific NetId, the default is 1.". Is this ok then?